src/components/screens/editor/object-key-conversion.tsx

```diff
@@ -0,0 +1,19 @@
+import { IntentTrainingMaterials, TrainingMaterialType } from "../../../generated/client/models";
```

Comment:
This file should be in src/utils/, and should be named editor.ts.

Reply:
Moved and switched name into editor.ts